lib/node.js

```diff
@@ -456,32 +494,62 @@ Node.prototype._initializeChain = function() {
 };
 
 Node.prototype.getServices = function() {
-  var defaultServices = {
-    'bitcoind': [],
-    'db': ['bitcoind'],
-    'chain': ['db']
-  };
-  return defaultServices;
+  var services = [
+    {
+      name: 'bitcoind',
+      dependencies: []
+    },
+    {
+      name: 'db',
+      dependencies: ['bitcoind'],
+    },
+    {
+      name: 'chain',
+      dependencies: ['db']
+    }
+  ];
+
+  services = services.concat(this._unloadedModules);
```

Comment:
This could be a for loop getting name and dependencies from the module, simplifying the config.